src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/SchemaCompareOptions.cs

```diff
@@ -0,0 +1,207 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.Dac;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts
+{
+    /// <summary>
+    /// Class to define schema compare and publish options. 
+    /// Keeping the order and defaults same as DacFx
+    /// The default values here should also match the default values in ADS UX
+    /// </summary>
+    public class SchemaCompareOptions
+    {
```

Comment:
Can we add a test the compare the properties from DacDeployOptions and SchemaCompareOptions, and fails when a DacDeployOptions is not present in SchemaCompareOptions? This we, can catch when properties are added when we pull in an updated nuget package.

Reply:
Done